codegen/src/main/twirl/templates/PlayJava/AkkaGrpcClientModule.scala.txt

```diff
@@ -3,29 +3,32 @@
  *@
 
 @(packageName: String, services: Seq[akka.grpc.gen.javadsl.Service])
-
 @akka.grpc.gen.Constants.DoNotEditComment
 package @{packageName};
 
-import play.api.Configuration;
-import play.api.Environment;
-import play.api.inject.Binding;
-import play.api.inject.Module;
+import com.typesafe.config.Config;
+import play.Environment;
+import play.inject.Binding;
+import play.inject.Module;
 import scala.collection.Seq;
+
 @services.map { service =>
 import @{service.packageName}.*;
 }
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Add this generated AkkaGrpcClientModule to play.modules.enabled
  * in your application.conf to have the available gRPC clients injectable
  */
 public class AkkaGrpcClientModule extends Module {
-  @@Override
-  public Seq<Binding<?>> bindings(Environment environment, Configuration configuration) {
-    return seq(
+  public List<Binding<?>> bindings(play.Environment environment, Config config) {
+    return Arrays.<Binding<?>>asList(
+        bindClass(GreeterServiceClient.class).toProvider(GreeterServiceClientProvider.class)
```

Comment:
We don't want to hard-code `Greeter` here, right?

Reply:
Yep!  Oops.